src/components/Home.jsx

```diff
@@ -5,36 +5,40 @@ import Leftside from './Leftside';
 import Main from './Main';
 import Rightside from './Rightside';
 
+import { IoClose } from 'react-icons/io5';
+
+import { Container, Section, Layout } from '../styles/stylesHome';
 
-import {
-  Container,
-  Section,
-  Layout
-} from "../styles/stylesHome";
- 
 const Home = (props) => {
   return (
     <Container>
-      {!props.user && <Redirect to = '/' />}
+      {!props.user && <Redirect to='/' />}
       <Section>
         <h5>
           <a> Looking for resources? - </a>
           <p> Find resources in record time with Maine's Reentry Resources!</p>
         </h5>
+        <IoClose
```

Comment:
What are you closing from here?

Reply:
this is a button to close the "looking for resources" message in the home page

Reply:
> this is a button to close the "looking for resources" message in the home page

I always have problems with close buttons, in this case I'm removing the parent, but the icon has a path tag inside.

Reply:
Oh that was just a general notice but it will be neet to dismiss it.